core/rint/test/TTabComTests.cxx

```diff
@@ -52,14 +55,44 @@ static std::string GetCompletions(const std::string& pattern)
    int pLoc = strlen(completed.get());
    std::ostringstream oss;
    ttc->Hook(completed.get(), &pLoc, oss);
-   return SortCompletions(oss.str());
+   return SortCompletions(oss.str(), ignore);
 }
 
-TEST(TTabComTests, Completion)
+TEST(TTabComTests, CompleteTH1)
 {
    // FIXME: The first call is unsuccessful due to a bug in the TTabCom::Hook
    // on some systems.
    GetCompletions("TH1");
+   std::string expected = "TH1 TH1C TH1D"
+#if defined(R__USE_CXXMODULES) && defined(R__HAS_DATAFRAME)
+      " TH1DModel"
```

Comment:
So in the end I believe that this change is hiding a real regression: https://root.cern/doc/master/structROOT_1_1RDF_1_1TH1DModel.html should not be one of the completion options as it is not on global scope.

Reply:
That is correct but see the discussion going from https://github.com/root-project/root/pull/5847/files#r468815354 onward. Essentially we currently regress when we have:
`using namespace ROOT::RDF` and type TH1<tab> and we do not complete the TH1DModel...

Reply:
> Essentially we currently regress when we have:
using namespace ROOT::RDF and type TH1 and we do not complete the TH1DModel

That's not a regression, it's a missing feature that would be great to have implemented!